vulture/core.py

```diff
@@ -121,9 +127,12 @@ def __hash__(self):
 
 class Vulture(ast.NodeVisitor):
     """Find dead code."""
-    def __init__(self, exclude=None, verbose=False, sort_by_size=False):
+    def __init__(self, exclude=None, verbose=False, sort_by_size=False,
+                 min_confidence=0.5):
         self.exclude = []
         self.sort_by_size = sort_by_size
+        self.min_confidence = float(min_confidence or 0.5)
```

Comment:
The default in the function arg is just to make it a non-optional param because we receive None anyways when we don't supply the `--min-confidence` flag.